docs/en/pid-issuing.rst

```diff
@@ -0,0 +1,598 @@
+    &code_challenge=E9Melhoa2OwvFrEMTJguCHaoeK1t8URWbuGJSstw-cM
+    &code_challenge_method=S256
+    &request=eyJhbGciOiJSUzI1NiIsImtpZCI6ImsyYmRjIn0.ew0KIC Jpc3MiOiAiczZCaGRSa3F0MyIsDQogImF1ZCI6ICJodHRwczovL3NlcnZlci5leGFtcGxlLmNvbSIsDQo gInJlc3BvbnNlX3R5cGUiOiAiY29kZSBpZF90b2tlbiIsDQogImNsaWVudF9pZCI6ICJzNkJoZFJrcXQz IiwNCiAicmVkaXJlY3RfdXJpIjogImh0dHBzOi8vY2xpZW50LmV4YW1...
+    &client_assertion_type=urn:ietf:params:oauth:client-assertion-type:jwt-key-attestation
+    &client_assertion=$WalletInstanceAttestation$ 
+
+The JWT payload of PAR is given as the following:
+
+.. code-block:: JSON
+
+    {
+    "response_type":"code",
+    "client_id":$thumprint-of-the-jwk-in-the-cnf-wallet-attestation$,
+    "state":"fyZiOL9Lf2CeKuNT2JzxiLRDink0uPcd",
+    "code_challenge":"E9Melhoa2OwvFrEMTJguCHaoeK1t8URWbuGJSstw-cM",
+    "code_challenge_method":"S256",
+    "authorization_details":[
+    {
+        "type":"openid_credential",
+        "format": "vc+sd-jwt",
+        "credential_definition": {
+            "type": ["eu.eudiw.pid.it"]
+        }
+    }
+    ],
+    "redirect_uri":"eudiw://start.wallet.example.org",
+    "client_assertion_type":"urn:ietf:params:oauth:client-assertion-type:jwt-key-attestation",
+    "client_assertion":$WalletInstanceAttestation$ 
+    }
+
+
+.. note::
+    **Federation Check:** PID Provider needs to check that the Wallet Provider is part of the federation and in addition it MUST verify Wallet Instance Attestation validity by checking its signature and the validity of the claims inside it. 
+
+**Step 7:** The PID Provider creates a new request URI representing this new authorization request and returns it to the Wallet Instance. 
+
+.. code-block:: 
+
+    HTTP/1.1 201 Created
+    Content-Type: application/json
+    Cache-Control: no-cache, no-store
+    
+    {
+        "request_uri":"urn:ietf:params:oauth:request_uri:bwc4JK-ESC0w8acc191e-Y1LTC2",
+        "expires_in": 60
+    }
+
+**Steps 8-9:** The Wallet Instance sends an authorization request to the PID Provider authorization endpoint.
+
+.. code-block:: 
+
+    GET /authorize?client_id=$thumprint-of-the-jwk-in-the-cnf-wallet-attestation$
+        &request_uri=urn%3Aietf%3Aparams%3Aoauth%3Arequest_uri%3Abwc4JK-ESC0w8acc191e-Y1LTC2 HTTP/1.1
+    Host: pid.it
+ 
+
+.. note::
+
+   **User Authentication and Consent:** The PID Provider performs the User authentication based on the requirements of eIDAS LoA High and asks the User for consent for the PID issuance. 
+
+**Steps 10-11:** The PID Provider sends an authorization code to the Wallet Instance. 
+
+.. note::
+
+    The Wallet Instance redirect URI is a universal or app link registered with the local operating system, so this latter will resolve it and pass the response to the Wallet Instance.
+ 
+.. code-block:: 
+
+    HTTP/1.1 302 Found
+    Location: eudiw://start.wallet.it?code=SplxlOBeZQQYbYS6WxSbIA&state=fyZiOL9Lf2CeKuNT2JzxiLRDink0uPcd
```

Comment:
even if it may appear weird, but I suggest "your suggestion" to have also "iss" here, as we agreed for CIE OIDC here
https://spid-cie-oidc-docs.vercel.app/authorization_endpoint.html#response

just to be sure that there might not be anyway to hijack the app locally to inject a fake response to drive the session to an adversary